This patch has now been merged upstream. Drop next update.
Commit: dc7ceddc502684cec0db8954125524d9da4c6036

This patch is needed for when cad/opencascade gets updated to 7.9.1.

Index: cMake/FindOCC.cmake
--- cMake/FindOCC.cmake.orig
+++ cMake/FindOCC.cmake
@@ -140,6 +140,10 @@ if(OCC_FOUND)
     TKHLR
     TKFeat
   )
+  if (OCC_VERSION_STRING VERSION_GREATER_EQUAL 7.9.0)
+    list(APPEND OCC_LIBRARIES TKExpress)
+  endif ()
+
   set(OCC_OCAF_LIBRARIES
     TKBin
     TKBinL
